# German translations for python-fedora.
# Copyright (C) 2010 ORGANIZATION
# This file is distributed under the same license as the python-fedora
# project.
#
# Laurin  <lineak@fedoraproject.org>, 2011.
# Mario Blättermann <mariobl@freenet.de>, 2011.
# Fabian Affolter <fab@fedoraproject.org>, 2009.
msgid ""
msgstr ""
"Project-Id-Version:  python-fedora\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2011-07-23 11:41-0700\n"
"PO-Revision-Date: 2011-07-23 17:24+0000\n"
"Last-Translator: toshio <a.badger@gmail.com>\n"
"Language-Team: de <LL@li.org>\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 0.9.6\n"

#: fedora/iterutils.py:36
msgid "fedora.iterutils is deprecated.  Use kitchen.iterutils instead"
msgstr ""

#: fedora/iterutils.py:54
msgid ""
"fedora.iterutils.isiterable is deprecated, use "
"kitchen.iterutils.isiterable instead"
msgstr ""

#: fedora/release.py:9
msgid "Python modules for interacting with Fedora Services"
msgstr "Python-Module zur Interaktion mit Fedora-Diensten"

#: fedora/release.py:10
msgid ""
"\n"
"The Fedora Project runs many different services.  These services help us "
"to\n"
"package software, develop new programs, and generally put together the "
"distro.\n"
"This package contains software that helps us do that.\n"
msgstr ""
"\n"
"Das Fedora-Projekt betreibt zahlreiche verschiedene Dienste. Diese "
"Dienste helfen beim\n"
"Paketieren von Software, Entwickeln neuer Programme und allgemein bei der"
" Zusammenstellung der Distribution.\n"
"Dieses Paket enthält Software, die uns dabei unterstützt.\n"

#: fedora/textutils.py:33
msgid "fedora.textutils is deprecated.  Use kitchen.text.converters instead"
msgstr ""

#: fedora/accounts/fas2.py:8
msgid ""
"fedora.accounts.fas2 has moved to fedora.client.fas2.  This location will"
" disappear in 0.4"
msgstr ""
"fedora.accounts.fas2 ist nun fedora.client.fas2. Dieser Ort wird in 0.4 "
"nicht mehr verfügbar sein"

#: fedora/client/__init__.py:89
msgid ""
"DictContainer is deprecated.  Use the Bunch class from python-bunch "
"instead."
msgstr ""
"DictContainer ist veraltet.  Verwenden Sie stattdessen die Bunch-Klasse "
"aus python-bunch."

#: fedora/client/baseclient.py:94 fedora/client/baseclient.py:215
#: fedora/client/baseclient.py:234 fedora/client/baseclient.py:246
msgid "session_cookie is deprecated, use session_id instead"
msgstr "session_cookie ist veraltet, verwenden Sie stattdessen session_id"

#: fedora/client/baseclient.py:112
#, python-format
msgid "Unable to load session from %(file)s"
msgstr "Sitzung konnte nicht aus %(file)s geladen werden"

#: fedora/client/baseclient.py:129
#, python-format
msgid "Unable to create %(dir)s: %(error)s"
msgstr "%(dir)s konnte nicht erstellt werden: %(error)s"

#: fedora/client/baseclient.py:141
#, python-format
msgid "Unable to write to session file %(session)s: %(error)s"
msgstr "Sitzungsdatei %(session)s konnte nicht geschrieben werden: %(error)s"

#: fedora/client/baseclient.py:164
#, python-format
msgid "No session cached for \"%s\""
msgstr "Keine Sitzung zwischengespeichert für »%s«"

#: fedora/client/baseclient.py:301
#, fuzzy, python-format
msgid "send_request() got an unexpected keyword argument \"%(arg)s\""
msgstr "send_request() ergab ein unerwartetes Schlüsselwort-Argument »%s«"

#: fedora/client/baseclient.py:305
msgid "send_request() got an unexpected keyword argument \"input\""
msgstr "send_request() ergab ein unerwartetes Schlüsselwort-Argument »input«"

#: fedora/client/baseclient.py:309
msgid "send_request() got an unexpected keyword argument"
msgstr "send_request() ergab ein unerwartetes Schlüsselwort-Argument"

#: fedora/client/baseclient.py:313
msgid "send_request(input) is deprecated.  Use send_request(req_params) instead"
msgstr ""
"send_request(input) ist veraltet.  Verwenden Sie stattdessen "
"send_request(req_params)"

#: fedora/client/baseclient.py:331
msgid ""
"Auth was requested but no way to perform auth was given.  Please set "
"username and password or session_id before calling this function with "
"auth=True"
msgstr ""
"Legitimierung wurde angefordert, aber keine Legitimierungsmethode "
"angegeben. Bitte legen Sie »username« und »password« oder »session_id« "
"fest, bevor Sie diese Funktion mit auth=True aufrufen"

#: fedora/client/bodhi.py:195
msgid "You must specify a username"
msgstr "Sie müssen einen Benutzernamen angeben"

#: fedora/client/bodhi.py:265
#, python-format
msgid "Reading from %s "
msgstr "Aus %s wird gelesen"

#: fedora/client/fas2.py:231
#, fuzzy, python-format
msgid "FAS server unable to retrieve group %(group)s"
msgstr "FAS-Server kann Gruppe %s nicht ermitteln"

#: fedora/client/fas2.py:363
msgid "key must be one of \"id\", \"username\", or \"email\""
msgstr "Schlüssel muss eines von »id«, »username«, oder »email« sein"

#: fedora/client/fas2.py:370
#, python-format
msgid "%(field)s is not a valid field to filter"
msgstr "%(field)s ist kein gültiges Feld zum Filtern"

#: fedora/client/fas2.py:422
msgid ""
"people_by_id() is deprecated and will be removed in 0.4.  Please port "
"your code to use people_by_key(key='id', fields=['human_name', 'email', "
"'username', 'bugzilla_email']) instead"
msgstr ""
"people_by_id() ist veraltet und wird in 0.4 entfernt. Bitte portieren Sie"
" Ihren Code zu people_by_key(key='id', fields=['human_name', 'email', "
"'username', 'bugzilla_email']) statt dessen"

#: fedora/client/fas2.py:604
msgid "FAS server unable to retrieve group members"
msgstr "FAS-Server kann Gruppenmitglieder nicht ermitteln"

#: fedora/client/fas2.py:626
msgid "FAS server unable to retrieve user information"
msgstr "FAS-Server kann Benutzerinformationen nicht ermitteln"

#: fedora/client/fasproxy.py:67
msgid "FasProxyClient() got an unexpected keyword argument 'session_as_cookie'"
msgstr ""
"FasProxyClient() ergab ein unerwartetes Schlüsselwort-Argument "
"»session_as_cookie«"

#: fedora/client/pkgdb.py:197
#, python-format
msgid ""
"We do not have enough information to create package %(pkg)s. Need version"
" owner."
msgstr ""

#: fedora/client/pkgdb.py:208
#, python-format
msgid "PackageDB returned an error creating %(pkg)s: %(msg)s"
msgstr "PackageDB gab einen Fehler bei der Erstellung von %(pkg)s zurück: %(msg)s"

#: fedora/client/pkgdb.py:231 fedora/client/pkgdb.py:287
#, python-format
msgid "Unable to save all information for %(pkg)s: %(msg)s"
msgstr "Speichern aller Informationen für %(pkg)s war nicht möglich: %(msg)s"

#: fedora/client/pkgdb.py:323
#, python-format
msgid "Collection abbreviation %(collection)s is unknown.  Use F, FC, EL, or OLPC"
msgstr ""

#: fedora/client/pkgdb.py:350
msgid ""
"collection and collection_ver are deprecated names for collctn_name and "
"collctn_ver respectively.  Ignoring the values given in them."
msgstr ""

#: fedora/client/pkgdb.py:356
msgid ""
"collection has been renamed to collctn_name.\n"
"Please start using the new name.  collection will go away in 0.4.x."
msgstr ""

#: fedora/client/pkgdb.py:362
msgid ""
"collection_ver has been renamed to collctn_ver.\n"
"Please start using the new name.  collection_ver will go away in 0.4.x."
msgstr ""

#: fedora/client/pkgdb.py:399
msgid ""
"collectn_list is a deprecated name for collctn_list.\n"
"Ignoring the value of collectn_list."
msgstr ""

#: fedora/client/pkgdb.py:404
msgid ""
"collectn_list has been renamed to collctn_list.\n"
"Please start using the new name.  collectn_list will go away in 0.4.x."
msgstr ""

#: fedora/client/pkgdb.py:478
msgid ""
"collectn is a deprecated name for collctn.\n"
"Ignoring the value of collectn."
msgstr ""

#: fedora/client/pkgdb.py:483
msgid ""
"collectn has been renamed to collctn.\n"
"Please start using the new name.  collectn will go away in 0.4.x."
msgstr ""

#: fedora/client/pkgdb.py:493
#, python-format
msgid "Collection shortname %(collctn)s is unknown."
msgstr ""

#: fedora/client/proxyclient.py:151
msgid "proxyclient.__init__:entered"
msgstr "proxyclient.__init__:entered"

#: fedora/client/proxyclient.py:160
msgid ""
"Returning cookies from send_request() is deprecated and will be removed "
"in 0.4.  Please port your code to use a session_id instead by calling the"
" ProxyClient constructor with session_as_cookie=False"
msgstr ""

#: fedora/client/proxyclient.py:167
msgid "proxyclient.__init__:exited"
msgstr "proxyclient.__init__:exited"

#: fedora/client/proxyclient.py:254
msgid "proxyclient.send_request: entered"
msgstr ""

#: fedora/client/proxyclient.py:263
msgid ""
"Giving a cookie to send_request() to authenticate is deprecated and will "
"be removed in 0.4. Please port your code to use session_id instead."
msgstr ""

#: fedora/client/proxyclient.py:273
msgid "username and password must both be set in auth_params"
msgstr "Benutzername und Password müssen in auth_params angegeben werden"

#: fedora/client/proxyclient.py:276
msgid ""
"No known authentication methods specified: set \"cookie\" in auth_params "
"or set both username and password in auth_params"
msgstr ""

#: fedora/client/proxyclient.py:365
#, python-format
msgid "Creating request %(url)s"
msgstr "Anfrage %(url)s wird erstellt"

#: fedora/client/proxyclient.py:367
#, python-format
msgid "Headers: %(header)s"
msgstr "Headers: %(header)s"

#: fedora/client/proxyclient.py:372
#, python-format
msgid "Data: %(data)s"
msgstr "Daten: %(data)s"

#: fedora/client/proxyclient.py:389
msgid "Authentication failed logging in"
msgstr "Legitimierung bei der Anmeldung ist gescheitert"

#: fedora/client/proxyclient.py:390
msgid ""
"Unable to log into server.  Invalid authentication tokens.  Send new "
"username and password"
msgstr ""
"Anmeldung am Server war nicht möglich. Ungültige Legitimierungs-Tokens. "
"Senden Sie Benutzername und Passwort erneut."

#: fedora/client/proxyclient.py:396
#, python-format
msgid "Attempt #%(try)s failed"
msgstr ""

#: fedora/client/proxyclient.py:403
msgid "Unknown HTTP Server Response"
msgstr "Unbekannte Antwort vom HTTP-Server"

#: fedora/client/proxyclient.py:425
#, fuzzy, python-format
msgid "Error returned from json module while processing %(url)s: %(err)s"
msgstr "Fehler von simplejson bei der Verarbeitung von %(url)s: %(err)s"

#: fedora/client/proxyclient.py:441
msgid "proxyclient.send_request: exited"
msgstr "proxyclient.send_request: Abbruch"

#: fedora/client/wiki.py:67
#, fuzzy, python-format
msgid "Login failed: %(data)s"
msgstr "Anmeldung gescheitert: %s"

#: fedora/client/wiki.py:89
#, python-format
msgid "From %(then)s to %(now)s"
msgstr "Von %(then)s bis %(now)s"

#: fedora/client/wiki.py:92
#, python-format
msgid "%d wiki changes in the past week"
msgstr "%d Wiki-Änderungen in der letzten Woche"

#: fedora/client/wiki.py:94
msgid ""
"Warning: Number of changes reaches the API return limit.\n"
"You will not get the complete list of changes unless\n"
"you run this script using a 'bot' account."
msgstr ""
"Warnung: Anzahl der Änderungen erreichte das Limit der API. Sie werden "
"keine vollständige Liste der Änderungen erhalten, es sei denn, Sie führen"
" dieses Skript mit einem »bot«-Konto aus."

#: fedora/client/wiki.py:104
msgid ""
"\n"
"== Most active wiki users =="
msgstr ""
"\n"
"== Aktivste Wiki-Benutzer =="

#: fedora/client/wiki.py:111
msgid ""
"\n"
"== Most edited pages =="
msgstr ""
"\n"
"== Am meisten bearbeitete Seiten =="

#: fedora/compat25/__init__.py:9
msgid ""
"fedora.pycompat25.defaultdict is deprecated, use "
"kitchen.pycompat25.collections.defaultdict instead"
msgstr ""

#: fedora/django/auth/models.py:50
msgid "Loading FAS groups..."
msgstr "FAS-Gruppen werden geladen …"

#: fedora/django/auth/models.py:56
msgid "Unable to load FAS groups. Did you set FAS_USERNAME and FAS_PASSWORD?"
msgstr ""
"FAS-Gruppen konnten nicht geladen werden. Haben Sie FAS_USERNAME und "
"FAS_PASSWORD gesetzt?"

#: fedora/django/auth/models.py:63
msgid ""
"FAS groups loaded. Don't forget to set FAS_USERNAME and FAS_PASSWORD to a"
" low-privilege account."
msgstr ""
"FAS-Gruppen wurden geladen. Vergessen Sie nicht, FAS_USERNAME und "
"FAS_PASSWORD zu setzen, um die Zugriffsrechte des Kontos zu beschränken."

#: fedora/tg/client.py:8
msgid ""
"fedora.tg.client has moved to fedora.client.  This location will "
"disappear in 0.4"
msgstr ""
"fedora.tg.client wurde fedora.client.  Dieser Ort wird in 0.4 nicht mehr "
"angezeigt"

#: fedora/tg/controllers.py:60
#, python-format
msgid "Welcome, %s"
msgstr "Wiillkommen, %s"

#: fedora/tg/controllers.py:69
msgid ""
"The credentials you supplied were not correct or did not grant access to "
"this resource."
msgstr ""
"Die von Ihnen angegebenen Anmeldeinformationen waren nicht korrekt oder "
"erlauben keinen Zugriff auf diese Ressource."

#: fedora/tg/controllers.py:72
msgid "You must provide your credentials before accessing this resource."
msgstr ""
"Sie müssen Ihre Anmeldeinformationen eingeben, bevor Sie auf diese "
"Ressource zugreifen können."

#: fedora/tg/controllers.py:75
msgid "Please log in."
msgstr "Bitte melden Sie sich an."

#: fedora/tg/controllers.py:100
msgid "You have successfully logged out."
msgstr "Sie haben sich erfolgreich angemeldet."

#: fedora/tg/tg1utils.py:100
msgid "url() expects a dictionary for query parameters"
msgstr "url() erwartet eine Datenquelle für Abfrageparameter"

#: fedora/tg/util.py:40
msgid ""
"fedora.tg.util is deprecated.  Switch to one of these instead:   TG1 "
"apps: fedora.tg.tg1utils   TG2 apps: fedora.tg.tg2utils.  This file will "
"disappear in 0.4"
msgstr ""

#: fedora/tg/identity/jsonfasprovider1.py:223
#: fedora/tg/identity/jsonfasprovider2.py:386
#, python-format
msgid "Error logging in %(user)s: %(error)s"
msgstr "Fehler beim Anmelden von %(user)s: %(error)s"

#: fedora/tg/identity/jsonfasprovider2.py:55
msgid ""
"Cannot enable ssl certificate auth via identity.ssl without setting "
"fas.usernamme and fas.password for authorization"
msgstr ""

#: fedora/tg/identity/jsonfasprovider2.py:152
#, python-format
msgid "jsonfasprovider, ssl, returned errors from send_request: %s"
msgstr ""

#: fedora/tg/identity/jsonfasprovider2.py:163
#, python-format
msgid "jsonfasprovider returned errors from send_request: %s"
msgstr ""

#: fedora/tg/templates/genshi/login.html:10
msgid "Log In"
msgstr "Anmeldung"

#: fedora/tg/templates/genshi/login.html:14
msgid "User Name:"
msgstr "Benutzername:"

#: fedora/tg/templates/genshi/login.html:17
msgid "Password:"
msgstr "Passwort:"

#: fedora/tg/templates/genshi/login.html:21
#: fedora/tg/templates/genshi/login.html:84
msgid "Login"
msgstr "Anmelden"

#: fedora/tg/templates/genshi/login.html:28
msgid "CSRF attacks"
msgstr "CSRF-Angriffe"

#: fedora/tg/templates/genshi/login.html:29
msgid ""
" are a means for a malicious website to make a request of another\n"
"        web server as the user who contacted the malicious web site.  The"
"\n"
"        purpose of this page is to help protect your account and this "
"server\n"
"        from attacks from such malicious web sites.  By clicking below, "
"you are\n"
"        proving that you are a person rather than just the web browser\n"
"        forwarding your authentication cookies on behalf of a malicious\n"
"        website."
msgstr ""

#: fedora/tg/templates/genshi/login.html:36
msgid "I am a human"
msgstr "Ich bin ein Mensch"

#: fedora/tg/templates/genshi/login.html:41
msgid "Forgot Password?"
msgstr "Passwort vergessen?"

#: fedora/tg/templates/genshi/login.html:42
msgid "Sign Up"
msgstr "Registrieren"

#: fedora/tg/templates/genshi/login.html:60
msgid "Welcome"
msgstr "Wllkommen"

#: fedora/tg/templates/genshi/login.html:82
msgid "You are not logged in"
msgstr "Sie sind nicht angemeldet"

#: fedora/tg/templates/genshi/login.html:89
msgid "CSRF protected"
msgstr "CSRF-geschützt"

#: fedora/tg/templates/genshi/login.html:91
msgid "Verify Login"
msgstr "Anmeldung bestätigen"

#: fedora/tg/templates/genshi/login.html:97
msgid "Logout"
msgstr "Abmelden"

#: fedora/tg/visit/jsonfasvisit1.py:107 fedora/tg/visit/jsonfasvisit2.py:136
#, python-format
msgid "updating visit (%s)"
msgstr "Aktualisiere Besuch (%s)"

#: fedora/wsgi/csrf.py:124
msgid "Creating CSRFProtectionMiddleware"
msgstr ""

#: fedora/wsgi/csrf.py:133
#, python-format
msgid "clean_environ(%s)"
msgstr "clean_environ(%s)"

#: fedora/wsgi/csrf.py:136
#, python-format
msgid "Deleting %(key)s from environ"
msgstr ""

#: fedora/wsgi/csrf.py:149
#, python-format
msgid "CSRFProtectionMiddleware(%(r_path)s)"
msgstr ""

#: fedora/wsgi/csrf.py:156
msgid "User supplied CSRF token matches environ!"
msgstr ""

#: fedora/wsgi/csrf.py:159
msgid "Clearing identity"
msgstr ""

#: fedora/wsgi/csrf.py:162
#, python-format
msgid ""
"Invalid CSRF token.  User supplied (%(u_token)s) does not match what's in"
" our environ (%(e_token)s)"
msgstr ""

#: fedora/wsgi/csrf.py:171
msgid "CSRF_AUTH_STATE; rewriting headers"
msgstr "CSRF_AUTH_STATE; Header werden neu geschrieben"

#: fedora/wsgi/csrf.py:177
#, python-format
msgid "response.location = %(r_loc)s"
msgstr ""

#: fedora/wsgi/csrf.py:249
#, python-format
msgid "CSRFMetadataProvider.add_metadata(%(r_path)s)"
msgstr ""

#: fedora/wsgi/csrf.py:255
#, python-format
msgid "session_id = %(s_id)r"
msgstr "session_id = %(s_id)r"

#: fedora/wsgi/csrf.py:261
msgid "Identity updated with CSRF token"
msgstr "Identität wurde mit CSRF-Token aktualisiert"

#: fedora/wsgi/csrf.py:264
msgid "Setting CSRF_AUTH_STATE"
msgstr "CSRF_AUTH_STATE wird gesetzt"

#: fedora/wsgi/csrf.py:274
msgid "Got HTTPFound(302) from repoze.who.application"
msgstr "HTTPFound(302) erhalten von repoze.who.application"

#: fedora/wsgi/csrf.py:287
#, python-format
msgid "Altered headers: %(headers)s"
msgstr "Geänderte Header: %(headers)s"

#: fedora/wsgi/csrf.py:290
#, python-format
msgid "Invalid session cookie %(s_id)r, not setting CSRF token!"
msgstr "Ungültiges Sitzungs-Cookie %(s_id)r, CSRF-Token nicht gesetzt!"

#: fedora/wsgi/csrf.py:300
#, python-format
msgid "%(token)s in GET"
msgstr "%(token)s in GET"

#: fedora/wsgi/csrf.py:308
#, python-format
msgid "%(token)s in POST"
msgstr "%(token)s in POST"

#: fedora/wsgi/faswho/faswhoplugin.py:123
msgid "Keep alive cache miss"
msgstr ""

#: fedora/wsgi/faswho/faswhoplugin.py:137
msgid "in identify()"
msgstr "in identify()"

#: fedora/wsgi/faswho/faswhoplugin.py:144
#, python-format
msgid "Request identify for cookie %(cookie)s"
msgstr ""

#: fedora/wsgi/faswho/faswhoplugin.py:168
msgid "In remember()"
msgstr "In remember()"

#: fedora/wsgi/faswho/faswhoplugin.py:181
msgid "In forget()"
msgstr "In forget()"

#: fedora/wsgi/faswho/faswhoplugin.py:188
#, python-format
msgid "Forgetting login data for cookie %(s_id)s"
msgstr "Anmeldedaten für Cookie %(s_id)s werden verworfen"

#: fedora/wsgi/faswho/faswhoplugin.py:205
msgid "In authenticate()"
msgstr "In authenticate()"

#: fedora/wsgi/faswho/faswhoplugin.py:229
msgid "Authentication failed, setting error"
msgstr "Legitimierung ist gescheitert, Einstellungsfehler"

#: fedora/wsgi/faswho/faswhoplugin.py:250
#, fuzzy
msgid "An unknown error happened when trying to log you in. Please try again."
msgstr ""
"Ein unbekannter Fehler trat bei Ihrer Anmeldung auf. Bitte versuchen Sie "
"es erneut."

#: fedora/wsgi/faswho/faswhoplugin.py:261
msgid "Metadata cache miss - refreshing metadata"
msgstr "Metadaten-Zwischenspeicher fehlt - Metadaten werden aktualisiert"

#: fedora/wsgi/faswho/faswhoplugin.py:292
msgid "In add_metadata"
msgstr "In add_metadata"

#: fedora/wsgi/faswho/faswhoplugin.py:296
msgid "Error exists in session, no need to set metadata"
msgstr ""

#: fedora/wsgi/faswho/faswhoplugin.py:306
#, python-format
msgid "Request metadata for cookie %(cookie)s"
msgstr ""

#: fedora/wsgi/test/controllers/root.py:31
msgid "Wrong credentials"
msgstr "Falsche Anmeldedaten"

#: fedora/wsgi/test/controllers/root.py:46
#, python-format
msgid "Welcome back, %s!"
msgstr "Willkommen zurück, %s!"

#: fedora/wsgi/test/controllers/root.py:56
msgid "We hope to see you soon!"
msgstr "Wir hoffen, Sie demnächst wieder begrüßen zu dürfen!"

#~ msgid ""
#~ "to_unicode was given \"%(obj)s\" which "
#~ "is neither a byte string (str) or"
#~ " a unicode string"
#~ msgstr ""

#~ msgid "non_string value, %(param)s, is not set to a valid action"
#~ msgstr ""

#~ msgid ""
#~ "to_bytes was given \"%(obj)s\" which is"
#~ " neither a unicode string or a "
#~ "byte string (str)"
#~ msgstr ""

#~ msgid "First argument must be callable"
#~ msgstr "Erstes Argument muss aufrufbar sein"

